rust/worker/src/server.rs

```diff
@@ -221,15 +276,15 @@ impl WorkerServer {
 
         for result_set in result {
             let mut proto_results = Vec::new();
-            for query_result in result_set {
+            for query_result in result_set.records {
                 let proto_result = chroma_proto::VectorQueryResult {
```

Comment:
somewhat out of scope but maybe it'd be useful to have `From<InternalRecord> for chroma_proto::VectorQueryResult`

Reply:
That's in the next PR